.eslintignore

```diff
@@ -2,4 +2,5 @@ node_modules
 dist
 .rpt2_cache
 packages/design/colors.js
+packages/design/icons
```

Comment:
Just to make sure I understand, I'll ask: why are we ignoring this here?

Reply:
`packages/design/src/icons` is where the actual code lives but `packages/design/icons` is for the generated css files for icons after build. This folder is also git [ignored](https://github.com/GetJobber/atlantis/blob/34420a8ac694fd719fe9a7a2dd1653689ef9e901/packages/design/.gitignore#L3).